components/micro-gateway-core/src/main/ballerina/Ballerina.toml

```
@@ -6,7 +6,7 @@ version= "3.1.0"
 target = "java8"
 
     [[platform.libraries]]
-    path = "../../../target/org.wso2.micro.gateway.core-3.1.0.jar"
+    path = "../../../target/org.wso2.micro.gateway.core-3.1.0-beta2-SNAPSHOT.jar"
```

Drop this change since we are getting ready for 3.1.0-rc
```suggestion
    path = "../../../target/org.wso2.micro.gateway.core-3.1.0.jar"
```